test/convert_test.go

```diff
@@ -1,147 +0,0 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package test
-
-import (
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"path/filepath"
-	"regexp"
-	"strings"
-	"testing"
-
-	"github.com/GoogleCloudPlatform/terraform-validator/converters/google"
-	"github.com/GoogleCloudPlatform/terraform-validator/version"
-)
-
-const samplePolicyPath = "sample_policies"
-
-var conversionTests = []struct {
-	name      string
-	assetType string
-}{
-	{"disk", "compute.googleapis.com/Disk"},
-	{"project", "cloudresourcemanager.googleapis.com/Project"},
-	{"project_billing_info", "cloudbilling.googleapis.com/ProjectBillingInfo"},
```

Comment:
Looks like this test got removed. More generally, how will we deal with cases (like this) where one resource (the project) actually generates two assets (project and project billing info)? Let's make sure we don't lose this test in the conversion.

Reply:
Billing is combined with the project test as it is handled via one single TF file.